demos/python_demos/rag_chatbot/README.md

```diff
@@ -0,0 +1,163 @@
+usage: compress_model.py [-h] --model
+                         {tiny-llama-1b-chat,red-pajama-3b-chat,llama-2-chat-7b,mpt-7b-chat,qwen-7b-chat,chatglm3-6b,mistral-7b,zephyr-7b-beta,neural-chat-7b-v3-1,notus-7b-v1,youri-7b-chat}
+
+Script to compress LLM model based on https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --model {tiny-llama-1b-chat,red-pajama-3b-chat,llama-2-chat-7b,mpt-7b-chat,qwen-7b-chat,chatglm3-6b,mistral-7b,zephyr-7b-beta,neural-chat-7b-v3-1,notus-7b-v1,youri-7b-chat}
+                        Select the LLM model out of supported list
+
+python compress_model.py --model ${SELECTED_MODEL}
+
+
+```
+Running this script will create new directories with compressed versions of the model with FP16, INT8 and INT4 precisions.
+The compressed models can be used in place of the original as they have compatible inputs and outputs.
+
+```bash
+ls  -1 | grep tiny-llama-1b-chat
+tiny-llama-1b-chat
+tiny-llama-1b-chat_FP16
+tiny-llama-1b-chat_INT4_compressed_weights
+tiny-llama-1b-chat_INT8_compressed_weights
+```
+
+> **NOTE** Applying quantization to model weights may impact the model accuracy. Please test and verify that the results are of acceptable quality for your use case.
+
+> **NOTE** On target devices that natively support FP16 precision (i.e. GPU), OpenVINO automatically adjusts the precision from FP32 to FP16. This improves the performance and typically does not impact accuracy. Original precision can be enforced with `ov_config` key:
+`{"INFERENCE_PRECISION_HINT": "f32"}`.
+
+## Download embedding model
+
+Download the model using `download_embedding_model.py` script:
+
+```bash
+python download_embedding_model.py --help
+usage: download_embedding_model.py [-h] --model {all-mpnet-base-v2,text2vec-large-chinese}
+
+Script to download LLM model based on https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --model {all-mpnet-base-v2,text2vec-large-chinese}
+                        Select the LLM model out of supported list
+
+python download_embedding_model.py --model all-mpnet-base-v2
+
+```
+The model will appear in `./all-mpnet-base-v2` directory.
+
+## Download test documents for knowledge base
+We will use single CSV file with [FIFA World Cups information](https://raw.githubusercontent.com/rangelak/fifa-stats/master/data/fifa-world-cup.csv).
+```bash
+wget -P documents/ https://raw.githubusercontent.com/rangelak/fifa-stats/master/data/fifa-world-cup.csv
+```
+
+## Deploy OpenVINO Model Server with Python Calculator
+
+Mount the `./documents` directory with knowledge base.
+Mount the `./llm_model` directory with the LLM model.  
+Mount the `./embed_model` directory with the document embedding model.  
+Mount the servable directory which contains:
```

Comment:
I wonder if we could mount it all in one go under /workspace?
We would have /workspace/documents, /workspace/llm_model, /workspace/embed_model, /workspace/servable
and then the docker run command would be smaller (a bit easier to read)